indico/htdocs/js/utils/forms.js

```diff
@@ -78,8 +97,37 @@
         }).on('change input', function() {
             var $this = $(this);
             var untouched = $this.serialize() == $this.data('initialData');
+            var $cornerMessage = $('.corner-message');
             $this.find('[data-disabled-until-change]').prop('disabled', untouched);
             $this.closest('form').data('fieldsChanged', !untouched);
+            if ($this.find('[data-save-reminder-message]').length) {
+                if (!isElementInView($this.find('[data-save-reminder-message]'))
+                && !untouched && $cornerMessage.length == 0) {
+                    showSaveCornerMessage($this);
+                }
+                else if (untouched) {
+                    $cornerMessage.fadeOut(300, function() {
+                        $(this).remove();
+                    });
+                }
+            }
+        });
+
+        // Remove corner message when 'Save' button is visible on screen
+        $(window).scroll(function() {
+            var $form = forms.find('[data-save-reminder-message]').closest('form')
+            if ($form.length) {
+                var $cornerMessage = $('.corner-message');
+                var untouched = $form.serialize() == $form.data('initialData');
+                if (isElementInView($form.find('[data-save-reminder-message]'))) {
+                    $cornerMessage.fadeOut(300, function() {
+                        $(this).remove();
+                    });
```

Comment:
Since you have a `showSaveCornerMessage`, why not a `hideSaveCornerMessage` as well with the above 3 lines? They are repeated anyway a bit above! A function would be useful imo.